### DISCUSSION TITLE
How to Manipulate Individual Cells in a Table Using the Designer?

### DISCUSSION THREAD

**Discussion by BeneUX** [pdfme/pdfme#608] [Q&A]
Hello,

I would like to know if it is possible to manipulate individual cells within a table using the Designer. If so, could you please explain how to achieve this?

Thank you for your help!

**Comment by hand-dot** [comment]
Hi @BeneUX 

Like this? or more advanced manipulation?

https://github.com/user-attachments/assets/3ab1b47d-aa1e-4072-a3e5-0f586cd54cec
